packages/grafana-data/src/transformations/fieldReducer.ts

```diff
@@ -232,6 +230,13 @@ export const fieldReducers = new Registry<FieldReducerInfo>(() => [
     description: 'Percentage difference between first and last values',
     standard: true,
   },
+  {
+    id: ReducerID.allValues,
+    name: 'All values',
+    description: 'Returns an array with all values',
+    standard: false,
+    reduce: (field: Field) => ({ allValues: field.values.toArray() }),
```

Comment:
:thinking: we will want to be careful about adding this here -- the general idea for a reducer is that it creates a single value out of many.  By adding this option to the registry, we also add it to everything else that expects a single value.  That may be OK... but need to think on it

Reply:
it is a single value :) just an array that is the value. But yes, one idea would be to mark this reducer as "returnsArray" or something so we can optionally include it in StatsPicker only for scenarios when it is supported